src/site/content/en/blog/get-installed-related-apps/index.md

```diff
@@ -141,13 +152,68 @@ your domain. Be sure to include the escaping characters.
 </string>
 ```
 
-### Test for the presence of your native app {: #test-native }
+Finally, publish your updated Android app to the Play store.
+
+### Define the relationship to an installed PWA {: #relationship-web }
+
+{% Aside %}
+**Coming Soon!**
+Expected in Chrome 81, websites can check if their PWA, served from the same
```

Comment:
81 beta is already out. It seems like we'd know already if this were shipping.

Reply:
Yes it is, you can try it out on backgroundfetch.com after installing the WebAPK

Reply:
Please change to 'Starting in Chrome 81'.